__tests__/helpers.js

```diff
@@ -0,0 +1,20 @@
+const consoleStubHandler = {
+  get: (target, prop) => {
+    if (!(prop in target) && prop in console) {
+      const spy = jest.spyOn(console, prop).mockImplementation(() => {});
+      target[prop] = spy;
+    }
+
+    return target[prop];
+  },
+};
+
+module.exports.silenceConsole = fn => {
+  const potentialSpies = new Proxy({}, consoleStubHandler);
```

Comment:
I regret trying to be cute here. I realized you can't use this and not write an expectation, since you have to call the dynamic getter.